source/material_model/rheology/visco_plastic.cc

```diff
@@ -453,6 +453,20 @@ namespace aspect
               composition_mask.set(i,false);
           }
 
+        // These are the compositional fields used to track deposition depth and
+        // sediment age of the sediments deposited by the FastScape plugin.
+        // ToDo: insert check that these fields are only set as a mask when fastscape is used?
```

Comment:
It's probably fine without checking, I don't think there is any instance where someone would set these fields and want them included in the rheology calculations.